src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -99,6 +103,13 @@ private static void aggregateBlameAuthorInfo(RepoConfiguration config, FileInfo
                 author = Author.UNKNOWN_AUTHOR;
             }
 
+            if (config.shouldIncludeLastModifiedDate()) {
+                // convert the commit date from the author time zone to cli-specified timezone
+                Date convertedCommitDate = TimeUtil.getSystemDateFromZonedDate(new Date(commitDateInMs),
```

Comment:
Should the method be `getZonedDateFromSystemDate`?

Reply:
@dcshzj may i get ur previous rationale in naming the method?

Reply:
Yes, I think it might be more appropriate to rename it to `getZonedDateFromSystemDate`. The Javadoc comment will need to be updated accordingly.